.github/ISSUE_TEMPLATE/package.yml

```
@@ -0,0 +1,63 @@
+name: Packages
+description: Report problems and suggest ideas for the packages
+labels: [package]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `package` label][package-issues] (including
+        closed issues) before you continue.
+
+
+        [package-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Apackage+is%3Aopen
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [Improvement, Bug, Other]
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Which packages do this issue concern?
+      options: [NPM, Packagist]
+      multiple: true
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Simple Icons Version
+      placeholder: "Example: 5.11.0"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Other Software
+      description: >-
+        Software with versions if known (Node.js & NPM, PHP & Packagist,
+        Browser). If you are unsure, enter "None."
```

I think we can improve on the wording here, what do you think about:

```suggestion
        The software that you are using the package with (Node.js & NPM, PHP & Packagist,
        Browser(s), other) and their versions. You can put "None" if you are unsure.
```